app/src/main/java/com/github/se/icebreakrr/ui/profile/OtherProfileView.kt

```diff
@@ -100,39 +120,71 @@ fun OtherProfileView(
                 sendRequest = false
               }
             }
-            InfoSection(profile, tagsViewModel)
-
-            // Add spacer for some padding
-            Spacer(modifier = Modifier.height(BUTTON_VERTICAL_PADDING))
-
-            // Already met button
-            Button(
-                onClick = {
-                  if (isNetworkAvailableWithContext(context)) {
-                    profilesViewModel.addAlreadyMet(profile.uid)
-                    meetingRequestViewModel.removeChosenLocalisation(profile.uid)
-                    Toast.makeText(context, R.string.Already_Met_Button_Success, Toast.LENGTH_SHORT)
-                        .show()
-                    profilesViewModel.getSelfProfile {}
-                    navigationActions.goBack()
-                  } else {
-                    showNoInternetToast(context = context)
+
+            // Scrollable content
+            Column(modifier = Modifier.fillMaxSize().verticalScroll(rememberScrollState())) {
+              InfoSection(profile, tagsViewModel)
+
+              // Add spacer for some padding
+              Spacer(modifier = Modifier.height(BUTTON_VERTICAL_PADDING))
+
+              // Ai button
+              Button(
+                  onClick = {
+                    bottomSheetVisible = true
+                    aiViewModel.findDiscussionStarter()
+                  },
+                  colors =
+                      ButtonDefaults.buttonColors(
+                          containerColor = MaterialTheme.colorScheme.primary),
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(BUTTONS_HORIZONTAL_PADDING)
+                          .align(Alignment.CenterHorizontally)) {
```

Comment:
I think you should precise `.padding(horizontal = BUTTONS_HORIZONTAL_PADDING)`else there is a huge space between the two buttons

Reply:
Thank you so much !